logos/logos-ui/src/app/features/model-error-report/model-performance-details.scss

```
@@ -0,0 +1,411 @@
+  top: 2px;
+  display: grid;
+  justify-items: center;
+  gap: 3px;
+  transform: translateX(-50%);
+  color: rgb(var(--color-typography-500));
+
+  i {
+    width: 4px;
+    height: 4px;
+    border-radius: 50%;
+    background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)